cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,31 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
+  const randomNumber = Math.random().toString().slice(2, 5);
+  const username = `test${randomNumber}`;
+  const email = `${username}@gmail.com`;
+  const password = 'Testpass1!';
```

Comment:
These variables are used in a single test case. It would be better to move them inside the scope of the test case to improve readability and maintainability.